test/chpldoc/compflags/version/projectVersion.no-tag.genindex.doc.good

```diff
@@ -0,0 +1,209 @@
+
+
+
+<!DOCTYPE html>
```

Comment:
I'm a little worried about these included files.  A lot of them have the potential to change when the sphinx version changes, meaning we'd constantly have to update them.  It would be better to only include the ones that reference the version, and add a prediff script to remove things from the .tmp file that could adjust